### ISSUE TITLE
Code illustrating "/tikz/ever pic" is not related to pic mechanics.

### ISSUE THREAD

**Issue by KZiemian** [pgf-tikz/pgf#821]
Version: 3.1.4b.

Section "18.2 The Pic Syntax", pages 262.

Code that should illustrate usage of "/tikz/every pic" is
```
\begin{tikzpicture}[every node/.style={draw}]
\draw (0,0) node {A} -- (1,1) node {B};
\end{tikzpicture}
```
have no relation to pic and is probably copy from chapter on the nodes.

**Comment by hmenke** [comment]
I'm pretty sure I fixed that one in 3.1.5, so please update your PGF and see if it's still there.  The current version is 3.1.5b.  You can also browse the latest manual online at https://pgf-tikz.github.io/pgf/pgfmanual.pdf

**Comment by hmenke** [comment]
Already fixed in 6ccafffa00bdb8f55e4f696325b202d37d3f1f6b
